src/LibraryManager/Providers/Unpkg/UnpkgProvider.cs

```diff
@@ -37,7 +37,9 @@ public UnpkgProvider(IHostInteraction hostInteraction)
 
         public ILibraryCatalog GetCatalog()
         {
-            return _catalog ?? (_catalog = new UnpkgCatalog(Id, LibraryNamingScheme, HostInteraction.Logger, WebRequestHandler.Instance));
+            var packageSearch = new NpmPackageSearch();
+            var infoCache = new NpmPackageInfoCache(packageSearch);
```

Comment:
Hm, so this cache will be separate from the one for the jsdelivr provider?

Reply:
How often is GetCatalog() called? If it's more than once, the cache will get recreated a lot.

On the other hand, how long do we want to maintain a cache of data that might never be used again?

Reply:
Now that I'm paying attention... packageSearch and infoCache are created every time this method is called, but _catalog is cached in the line below. The objects above are created for no reason if _catalog != null.

(Same applies to jsDeliver)

Reply:
Resolved with commit 8